content/blog/2017/12/2017-12-31-new-year.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Happy New Year!"
+tags:
+- community
+author: oleg_nenashev
+---
+
+Jenkins project congratulates all users and contributors with the New Year!
+Let's take a look at some changes this year.
+
+image:/images/post-images/2017-12-31-new-year/card.png[NewYear, role=center]
+
+## Highlights
+
+* We released major features like link:/projects/blueocean[BlueOcean]
+and link:/doc/book/pipeline/syntax/#declarative-pipeline[Declarative Pipeline]
+** These features offer a new user experience in Jenkins Web UI, powered by link:/doc/book/pipeline/[Jenkins Pipeline].
+* In Jenkins 2.54 we updated the Java minimal requirement to Java 8 (link:/blog/2017/04/10/jenkins-has-upgraded-to-java-8/[announcement]).
```

Comment:
Should we mention 2.60.1?

Reply:
I think it's fine in the current state, but I can add it if you prefer it

Reply:
I think it's enough. We'll add it if a question comes about it for instance. But I expect not. Thanks for writing this.